src/Engine/Manager/RenderManager.cpp

```diff
@@ -268,7 +268,7 @@ void RenderManager::RenderWorldEntities(World& world, const glm::mat4& viewMatri
             Entity* lightEntity = spotLight->entity;
             lightViewMatrix = glm::inverse(lightEntity->GetModelMatrix());
             ///Will use range 50.f on the projection, no support for spotlight lenght. 
```

Comment:
Not from your branch but, "length" - spelling error. Space after // and dot at end.